front/src/styles/scss/applications/operationalStudies/_nge.scss

```diff
@@ -0,0 +1,5 @@
+iframe {
```

Comment:
We probably want to better target the NGE iframe specifically (e.g. with a CSS class). As is, this applies to any iframe in OSRD.

Reply:
Done